### DISCUSSION TITLE
Welcome to FFNx Discussions - No support for 3rd party mod tools ( 7th Heaven, etc. )! - READ THIS BEFORE POSTING

### DISCUSSION THREAD

**Discussion by julianxhokaxhiu** [julianxhokaxhiu/FFNx#160] [General]
## 👋 Welcome!
  We’re using Discussions as a place to talk about issues around FFNx or ideas that could be later on moved on as working items ( Issues ). Feel free to use this area for anything related to:
  * Asking questions about FFNx
  * Share ideas
  * Engage with other community members
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  Please consider using this space for discussing anything related to FFNx and NOT for issues or bugs found. For that consider opening a new issue!

# NO SUPPORT FOR EXTERNAL MOD TOOLS!
**No support** will be provided when using FFNx with 7th Heaven, Junction VIII or other mod tools around it! When reporting issues, make sure you can reproduce them using FFNx in **vanilla mode**!

**Comment by MaxVicioOk** [comment]
what is the proper place to report problems with 7th heaven? in the whole forum it says that "in this section there is no support"

**Reply by julianxhokaxhiu** [reply]
Both Discord servers mentioned here do have a dedicated 7th Heaven support channel: https://github.com/julianxhokaxhiu/FFNx#discord
